pkg/infra/infra.go

```diff
@@ -229,12 +229,10 @@ var _ = Describe("[Feature:Machines] Managed cluster should", func() {
 		Expect(err).ToNot(HaveOccurred())
 		Expect(machines).ToNot(BeEmpty())
 
-		machine := machines[0]
-
-		By(fmt.Sprintf("deleting machine object %q", machine.Name))
-		err = framework.DeleteMachine(client, machine)
+		By(fmt.Sprint("deleting all machines"))
+		err = framework.DeleteMachines(client, machines...)
```

Comment:
how is this change related to this PR?

Reply:
It is the same thing, as in comment https://github.com/openshift/cluster-api-actuator-pkg/pull/160/files#r431705540, will completely exclude this and rebase after merging. Letft it here in hope to see CI jobs passing.